gpii/node_modules/flowManager/test/shared/BrowserChannelTestDefs.js

```diff
@@ -0,0 +1,466 @@
+/**
+ * GPII Flow Manager Browser Channel Tests
+ *
+ * Copyright 2013 OCAD University
+ * Copyright 2015 Emergya
+ * Copyright 2015 Raising the Floor - Internawional
```

Comment:
You have spotted my Easter Egg based on a transcription of a hostel sign seen in China in 2005, as attested on this blog : P 
http://transasia.blogg.se/2010/may/